### PR TITLE
Right component fixes #47

### LINKED ISSUE #47: Update right side component

### PR DESCRIPTION
**Pull request by maaz-bin-asad**
- Added additional required text in apply section.
- Removed newsletter part.
- Shuffled the component buttons as mentioned in slides.

Screenshot (273)

### CODE REVIEW

File: components/Apply.js [hunk 1]
```diff
@@ -22,22 +22,23 @@ const Apply = () => {
...
-            contribution at any time.
+            By supporting students, you are becoming an active participant in
+            building their careers. All donations and contributions directly sponsor the attempts of talented students to 
+            secure admission in an MS/PhD program at a university of international repute. You can easily cancel or upgrade your contribution
+            at any time.
           </p>
         </div>
         <button className="px-2 py-2 mt-8 w-full bg-red-850 text-sm text-white text-center">
-          SSGSA results (2021-2022) announce
+          The Aligarh Muslim University
```
Comment (pnijhara) [inline components/Apply.js:32]: This should be a link to https://amu.ac.in

File: components/Apply.js [hunk 2]
```diff
@@ -22,22 +22,23 @@ const Apply = () => {
...
         </button>
         <button className="px-2 py-2 mt-3 w-full bg-red-850 text-sm text-white text-center">
-          Pledge your support to SSGSA
+          SSGSA results (2021-2022) announced
```
Comment (pnijhara) [inline components/Apply.js:35]: Look at the original ssgsa.us website. And add the document to which this button is pointing.

File: components/Apply.js [hunk 3]
```diff
@@ -22,22 +22,23 @@ const Apply = () => {
...
         </button>
         <button className="px-2 py-2 mt-3 w-full bg-red-850 text-sm text-white text-center">
-          Federation of Aligarh Alumni Associations(FAAA)
+          Pledge your support to SSGSA
```
Comment (pnijhara) [inline components/Apply.js:38]: Update this link as well. Take help from ssgsa.us

File: components/Apply.js [hunk 4]
```diff
@@ -22,22 +22,23 @@ const Apply = () => {
...
         </button>
         <button className="px-2 py-2 mt-3 w-full bg-red-850 text-sm text-white text-center">
-          The Aligarh Muslim University
+          Federation of Aligarh Alumni Associations(FAAA)
```
Comment (pnijhara) [inline components/Apply.js:41]: Add the link

File: components/Apply.js [hunk 5]
```diff
@@ -22,22 +22,23 @@ const Apply = () => {
...
         </button>
         <button className="px-2 py-2 mt-3 w-full bg-red-850 text-sm text-white text-center">
           Join our Facebook Group
```
Comment (pnijhara) [inline components/Apply.js:44]: Add the Facebook group link.

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: components/Apply.js (hunk 1 above)
Comment (maaz-bin-asad) [reply components/Apply.js:33]: Updated. Please review

File: components/Apply.js [hunk 6]
```diff
@@ -22,44 +22,45 @@ const Apply = () => {
...
+            at any time.
           </p>
         </div>
+        <a href="https://amu.ac.in/">
         <button className="px-2 py-2 mt-8 w-full bg-red-850 text-sm text-white text-center">
-          SSGSA results (2021-2022) announce
+          The Aligarh Muslim University
         </button>
+        </a>
+        <a href="http://ssgsa.us/uploads/PressRelease-SSGSA-2021-22.pdf">
```
Comment (pnijhara) [inline components/Apply.js:36]: This shouldn't be done this way. You have to upload the document in this project and then the reference to be given.

File: components/Apply.js [hunk 7]
```diff
@@ -22,44 +22,45 @@ const Apply = () => {
...
         <button className="px-2 py-2 mt-3 w-full bg-red-850 text-sm text-white text-center">
-          Pledge your support to SSGSA
+          SSGSA results (2021-2022) announced
         </button>
+        </a>
+        <a href="https://docs.google.com/forms/d/e/1FAIpQLSfdZypeiKqTu5AR573bFuVxmw_rZHgWWDAt--8fkWIosNH_-w/viewform">
```
Comment (pnijhara) [inline components/Apply.js:41]: This will work

File: components/Apply.js [hunk 8]
```diff
@@ -22,44 +22,45 @@ const Apply = () => {
...
-          The Aligarh Muslim University
+          Federation of Aligarh Alumni Associations(FAAA)
         </button>
+        </a>
+        <a href="https://www.facebook.com/groups/ssesa/">
         <button className="px-2 py-2 mt-3 w-full bg-red-850 text-sm text-white text-center">
           Join our Facebook Group
         </button>
+        </a>
+        <a href="http://ssgsa.us/assets/SSGSA_Convention_2020_Press_Release.pdf">
```
Comment (pnijhara) [inline components/Apply.js:56]: Change this

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: components/Apply.js (hunk 6 above)
Comment (pnijhara) [reply components/Apply.js:36]: Also, for such anchor tags, give target as blank. It is good to open documents in other tabs.

File: components/Apply.js (hunk 8 above)
Comment (maaz-bin-asad) [reply components/Apply.js]: Updated. Please review

**Pull request merged**